api/RestfulService.php

```diff
@@ -67,6 +67,22 @@ public static function set_default_curl_options($optionArray) {
 	}
 
 	/**
+	 * Parses cURL options defined in YML configuration.
+	 *
+	 * @return array
+	 */
+	public static function get_default_curl_options() {
+		$rawOptions = Config::inst()->get('RestfulService', 'default_curl_options');
```

Comment:
All set on that: https://github.com/patricknelson/silverstripe-framework/commit/5e8d166b48ba7037f609e822ccbe2554834cf804#diff-e0ab4000778eb49720f320171895f8d3R75

Also note, in the interest of extensibility, the *other* references which must be calling `Config::inst()->update()` directly (since there is no ability to call `->update()` via `static::config()`) were also updated to use `get_called_class()` here: https://github.com/patricknelson/silverstripe-framework/commit/5e8d166b48ba7037f609e822ccbe2554834cf804#diff-e0ab4000778eb49720f320171895f8d3R56

In the same vogue (extensibility), I abstracted the instantiation of the `RestfulService_Response` class to a method since SS unfortunately doesn't use IoC (as far as I know) here, in case the developer does ever extend the class (i.e. the point to the above changes): https://github.com/patricknelson/silverstripe-framework/commit/5e8d166b48ba7037f609e822ccbe2554834cf804#diff-e0ab4000778eb49720f320171895f8d3R371 -- that also impacted this line as well: https://github.com/patricknelson/silverstripe-framework/commit/5e8d166b48ba7037f609e822ccbe2554834cf804#diff-e0ab4000778eb49720f320171895f8d3R655